sermant-backend/src/main/java/io/sermant/backend/entity/heartbeat/HeartbeatMessage.java

```diff
@@ -54,6 +54,14 @@ public class HeartbeatMessage {
 
     private Map<String, PluginInfo> pluginInfoMap = new HashMap<>();
 
+    private String appName;
+
+    private String artifact;
+
+    private String processId;
+
+    private boolean dynamicMount;
```

Comment:
dynamicMount is aimed to Agent status？Mount is not unified noun，we use dynamicInstall

Reply:
done